pkgs/top-level/all-packages.nix

```
@@ -15824,24 +15824,24 @@ in
         src = fetchFromGitHub {
           owner = "KhronosGroup";
           repo = "SPIRV-Tools";
-          rev = "323a81fc5e30e43a04e5e22af4cba98ca2a161e6";
-          sha256 = "1kwyh95l02w3v1ra55c836wayzw8d0m14ab7wf0ynhhyp3k2p9hv";
+          rev = "e128ab0d624ce7beb08eb9656bb260c597a46d0a";
+          sha256 = "0jj8zrl3dh9fq71jc8msx3f3ifb2vjcb37nl0w4sa8sdhfff74pv";
         };
       });
       spirv-headers = spirv-tools.overrideAttrs (_: {
```

This is a typo right? It should be:

```suggestion
      spirv-headers = spirv-headers.overrideAttrs (_: {
```